setup.cfg

```
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.12.0
+current_version = 2.12.1
```

I think you should bump the minor version since this is additive functionality and not a minor bug fix.

```suggestion
current_version = 2.13.0
```